authors/apps/authentication/models.py

```diff
@@ -1,18 +1,20 @@
 import jwt
+import os
```

Comment:
The changes in this file do not affect the current task am I right??

Reply:
YEAH 😄

Reply:
I would suggest you remove it then 😄.